packages/storybook/stories/Stateful Client/Adapters.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { Canvas, Description, Heading, Props, Source, Title } from '@storybook/addon-docs/blocks';
+
+<Meta id="statefulclient-reacthooks-setup" title="Stateful Client/React Hooks/Adapters" />
+
+# Adapters for Composites
+
+Composites deliver turn-key communication experiences for developers.
+They make it easy for developers to add end-to-end communication experiences to their apps with only a couple lines of code.
+To power Composites, the `communication-react` library provides `Adapters`, one for calling and one for chat Composites.
+These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like usePropsFor, to provide developers a single interface to configure Composites.
+
+## Setting Up Adapters
+
+To setup the adapters, you will need to provide the adapter with key metadata required.
+This includes most importantly an access token for your Azure Communication Services resource and context information for the call or chat thread you want to join.
+To create the adapters we will use the `createAzureCommunicationCallAdapter` and `createAzureCommunicationChatAdapter` helper methods.
+This methods are asynchronous so we will will `useEffect` to create them.
+
+### Calling Adapters
+
+To initialize the calling adapter, you will be required to provide a call locator with the information of the group or meeting you are trying to join.
+Group Id for Azure Communication Services take the shape of GUID generated by the developer to identify a call inside their tenant.
+Teams Meeting information can be acquired through [Microsoft Graph](https://docs.microsoft.com/en-us/graph/overview).
+
+```typescript
+import { CallAdapter, createAzureCommunicationCallAdapter } from '@azure/communication-react';
+
+const [callAdapter, setCallAdapter] = useState<CallAdapter>();
+const accessToken = 'INSERT ACCESS TOKEN';
+const displayName = 'INSERT DISPLAY NAME FOR LOCAL USER';
+const callLocator = { groupId: 'INSERT GROUP ID' };
+//CallLocator for Teams Meeting = const callLocator = {meetingLink: 'INSERT MEETING LINK'};
```

Comment:
Last time I tried, `meetingLink` was only available in the adapter beta package. Feedback I received is that currently beta vs latest is very confusing on NPM. Developers expect that `1.1.0` is a final version of `1.1.0-beta.2`, where currently only versions with the `-beta` prefix have the Teams interop typing (and other features) included.

Would make sense to have this properly documented here. That they need to use a beta package if they want to use Teams interop.

Reply:
The concepts of the stateful client + composites are all beta related concepts. There is nothing here for GA just yet. I think we can find a way to tag this type of feature as beta at the top and hopefully that is sufficient.